dbt/adapters/clickhouse/relation.py

```diff
@@ -118,14 +118,13 @@ def create_from(
             if schema == relation_config.source_name and relation_config.database:
                 schema = relation_config.database
 
+        if relation_config.config.get("force_on_cluster") in ("True", "true"):
+            can_on_cluster = True
```

Comment:
What if `cluster` is not set in config but `force` is enabled? What should happen? Should we error or set `can_on_cluster` to False?